Makefile

```
@@ -17,6 +17,7 @@ else ifeq ($(ARCH),RV64)
 else
   $(error '$(ARCH)' is not a valid architecture, must be one of: RV32, RV64)
 endif
+SAIL_XLEN += riscv_xlen.sail
 
```

```suggestion

SAIL_XLEN += riscv_xlen.sail
```

Keeping all of these grouped?